package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "crispy-toolkit",
```

Comment:
How about `crispy-cli` as the name? It would more transparent, but maybe you plan additional responsibilities to call it `toolkit`?

Reply:
I was thinking about two different usages from cli and as a package.
let's use `crispy-cli`. Thinking about having CLI tool mutch better than a lot of libraries for different programming languages

BTW, looks like we are able to use either `crispy-cli` or `crispy-toolkit`
https://www.npmjs.com/search?q=crispy-cli
https://www.npmjs.com/search?q=crispy-toolkit